packages/nextjs/pages/builders/0x2F142f9F52af0845B7b8b1353E349Ca1Cb8EFE92.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { appChains } from "~~/services/web3/wagmiConnectors";
+
+const PapaWheelie = () => {
+  const [ensData, setEnsData] = useState<Record<string, string | undefined>>({});
+  const nameTitle = "P A P A W H E E L I E";
+  const ensName = "papawheelie.eth";
+
+  useEffect(() => {
+    const client = appChains.publicClient({
+      chainId: 1,
+    });
+    async function fetchEnsTexts() {
+      const keys = ["com.twitter", "bio", "com.discord", "com.youtube", "com.github", "avatar"];
+      try {
+        const texts = await Promise.all(
+          keys.map(key =>
+            client
+              .getEnsText({
+                name: "papawheelie.eth",
+                key: key,
+              })
+              .then(text => ({ key, text })),
+          ),
+        );
+
+        const newData = texts.reduce((acc, { key, text }) => {
+          acc[key] = text ?? undefined;
+          return acc;
+        }, {} as Record<string, string | undefined>);
+
+        setEnsData(newData);
+      } catch (error) {
+        console.error("Error fetching ENS texts:", error);
+      }
+    }
+
+    fetchEnsTexts();
+  }, []);
+
+  return (
+    <div className="flex h-lvh flex-grow items-center justify-center px-4">
+      <div className="mx-auto max-w-3xl rounded-md shadow-2xl shadow-indigo-500/40 md:shadow-indigo-500/40 md:shadow-2xl">
+        <div className="flex flex-col md:flex-row items-center items-stretch justify-center gap-4">
+          <div className="md:w-1/3 flex justify-center md:justify-start">
+            <img
```

Comment:
[Lint fails](https://github.com/BuidlGuidl/batch1.buidlguidl.com/actions/runs/7565873726/job/20602308172?pr=12) since we are using normal `img` tag instead of Next `Image`.  I think we can ignore it for simplicity and not needing to update next.config with url patterns.

You could just add this comment `// eslint-disable-next-line @next/next/no-img-element` above `<img`  to ignore the warning 

Thanks @mrpapawheelie 🙌